mxcubecore/HardwareObjects/TangoShutter.py

```diff
@@ -31,19 +31,27 @@
   <command type="tango" name="Open">Open</command>
   <command type="tango" name="Close">Close</command>
   <channel type="tango" name="State" polling="1000">State</channel>
-  <values>{"open": "OPEN", "cloded": "CLOSED", "DISABLE" : "DISABLE"}</values>
+  <values>{"OPEN": "MYOPEN", "NEWSTATE": [HardwareObjectState.BUSY, "MYSTATE"]}</values>
```

Comment:
If this is meant to be JSON, it is not valid JSON.

Maybe it is supposed to read:

```suggestion
  <values>{"OPEN": "MYOPEN", "NEWSTATE": ["HardwareObjectState.BUSY", "MYSTATE"]}</values>
```

Reply:
Thanks for pointing that out. That part of the commentry slipped through. I didn't need to use a list for the configuration but I will look into it and do some tests with a configuration that takes a list.

Reply:
This could be changed to
`<values>{"OPEN": "MYOPEN", "NEWSTATE": ["MYSTATE", "BUSY"]}</values>`